### PR TITLE
refined Compute adoption procedure

### PR DESCRIPTION
**Pull request by klgill**
https://issues.redhat.com/browse/RHOSPDOC-1719

### CODE REVIEW

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (klgill) [inline docs_user/modules/proc_adopting-the-compute-service.adoc:27]: @SeanMooney FYI: I removed most of the prerequisites because the customer should be following the guide in order, and they would have done all of this already. But if there’s a prerequisite that you think we should emphasize, please let me know.

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (klgill) [inline docs_user/modules/proc_adopting-the-compute-service.adoc:143]: @SeanMooney I’m not sure if we need to say “In a multi-cell setup, it should become indexed as the last cell instead.” This procedure is focused on a single cell configuration, so I suggest staying focused on that.

Comment (SeanMooney) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:143]: that is debatable. 
the Bogdan is currently working on the support for multiple compute cells

we are generally trying to remove the single cell and multi-cell disticiton form all of our docs

on of the open questions we are currently resolving is if we can have a common adoption procedure or not.


all OpenStack deployments have at least 2 cells 

cell-0 which is reserved for instance that could not be scheduled to any host and some other internal nova usage and a compute cell typically called "cell-1" or sometimes "default"

when i refer to a comptue cell i mean a cell other then cell-0
or endgoal is to have one procedure for both cases since we will no longer be maintaining a difference in our docs for a cloud that only has 2 cells (cell-0 and either cell-1 or default) and a cloud that has more the two cells.

Comment (SeanMooney) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:143]: @klgill for context this is the current work in flight to update the procedure to work with multiple compute cells https://github.com/openstack-k8s-operators/data-plane-adoption/pull/517

Comment (bogdando) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:143]: I have discovered a regression in docs, which makes it disconnected from tests, see https://github.com/openstack-k8s-operators/data-plane-adoption/pull/490#issuecomment-2228606100

**Review by bogdando — CHANGES_REQUESTED** [review]
This guide refactoring should be done after fixing a regression discovered here https://github.com/openstack-k8s-operators/data-plane-adoption/pull/490#issuecomment-2228606100

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (bogdando) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:143]: regarding the multi-cell work in progress, I can make it rebasing on top of this, hopefully. Ideally though I agree this should be blocked on finishing the multi-cell guide in the PR that Sean has linked.

However, if this need to proceed now, feel free to just omit any confusing sections and statements related to multi-cell arch, as it was in flux by the time of writing, and still it is

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (bogdando) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:27]: I would prefer retaining the list of dependencies explicit

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (klgill) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:143]: @bogdando This procedure is intended to be published in the adoption guide async after GA. It will likely be published before Feature Release 1, when the multi-cell feature is finished. 
I'm inclined to do as you suggested: Remove any statements related to multi-cell and then merge. Let me know if anything else should be done.

Comment (jistr) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:143]: +1, i think this PR contains a bunch of refinements that are unrelated to the multi-cell situation? So ideally we wouln't prevent this from landing altogether.

Comment (klgill) [reply docs_user/modules/proc_adopting-the-compute-service.adoc]: FYI @SeanMooney @bogdando @jistr : I hid/removed all multi-cell references. Let me know if I missed anything, or if you have additional comments on this procedure in general. Thank you!

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (klgill) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:27]: **Note to peer reviewers**: I'd like a second opinion on listing prerequisites. If customers should be following the guide in order, do we need to list every procedure that they already should have done?

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (gregraka) [inline docs_user/modules/proc_adopting-the-compute-service.adoc:23]: Suggestion: use active voice:

You cannot run the metadata service in `cell0`.

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (gregraka) [inline docs_user/modules/proc_adopting-the-compute-service.adoc:117]: Looking at the Compute doc, it would seem that superconductor is two words:
https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html-single/scaling_deployments_with_compute_cells/index#assembly_multi-cell-overcloud-deployments_cellsv2

Unless this is the name of some file or actual process.

**Review by gregraka — APPROVED** [review]
@klgill solid effort. A few nits that you can take or leave!

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (SeanMooney) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:117]: we use it both ways 
the downstream docs are not the source of truth to reference
instead we should look to the upstream docs

https://github.com/openstack/nova/blob/master/doc/source/admin/cells.rst#api-database

we typically refer to this as one word

but we also spell it out as two 

https://github.com/openstack/nova/blob/1191a49a65dce694b12df26bb5316b0a816b278e/doc/source/admin/cells.rst?plain=1#L179

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (JoanneOFlynn2018) [reply docs_user/modules/proc_adopting-the-compute-service.adoc:27]: You could leave in just the first prerequisite to be explicit. 
Previous Adoption steps completed. 
Reword:
You have completed the previous Adoption steps.

File: docs_user/modules/proc_adopting-the-compute-service.adoc
Comment (JoanneOFlynn2018) [inline docs_user/modules/proc_adopting-the-compute-service.adoc:13]: As this is a prerequisite, the user should already have performed it. So reword to: 

```suggestion
* You have defined the following shell variables. Replace the following example values with the values that are correct for your environment:
```

**Comment by klgill** [comment]
recheck

**Pull request merged**
